pytest.ini

```diff
@@ -8,3 +8,4 @@ norecursedirs = .* docs completion refactor absolute_import namespace_package sc
 # fine as long as we are using `clean_jedi_cache` as a session scoped
 # fixture.
 usefixtures = clean_jedi_cache
+testpaths = test
```

Comment:
Ah, that starts to make sense. When I added the directory back, the failure about missing fixture returned and now it started making sense: `clean_jedi_cache` fixture is not found for doctests module running in `jedi` directory, because it is in a `conftest.py` in a  different directory.